### DISCUSSION TITLE
GHCR: see which tags refer to the same image

### DISCUSSION THREAD

**Discussion by kawaja** [community/community#26901] [Actions]
<p>It would be good to be able to see which tags in the container registry refer to the same image (e.g. <code>latest</code> is the same as <code>v0.5</code> and <code>v0.5.2</code> and <code>v0.4</code> is different). This could be by showing the SHA for the image, or some other method. <a class="hashtag" href="https://github.community/tag/ghcr">#<span>ghcr</span></a></p>

**Comment by clarkbw** [comment]
<p>This is coming alongside a new container landing page. Both on that page, and the versions page, it will show tags for each version instead of individual rows for each tag.</p>
<p>Check back later this week!</p>

**Comment by clarkbw** [comment]
<p>Staff ship went out today… should roll out this week.</p>

**Comment by clarkbw** [comment, marked as answer]
<p>New UI went out today!</p>
<p>Check out <a href="https://github.blog/changelog/2020-09-14-github-container-registry-new-landing-pages-with-repository-linking/">https://github.blog/changelog/2020-09-14-github-container-registry-new-landing-pages-with-repository-linking/</a></p>
